### ISSUE TITLE
Microsoft.AspNetCore.Authentication.JwtBearer 3.x is not compatible with netstandard2.1

### ISSUE THREAD

**Issue by CreepyGnome** [dotnet/aspnetcore#20414]
This is a breaking change for people who were using Microsoft.AspNetCore.Authentication.JwtBearer 2.2.0 which was netsteadard2.0.

I know people have complained about and you just said change the library to stop being .NET Standard, and I find that unacceptable as you never explain why you made this breaking change,  As .NET Standard should be where we try to make as many of our supporting libraries compatible with.

We have .NET Standard libraries that if we change them to a core library we have to do that all the way up the chain, and that is not acceptable.  

Was this breaking change made by accident and you could just as easily build and deploy Microsoft.AspNetCore.Authentication.JwtBearer as .NET Standard or was it on purpose and no one documented it in the breaking changes migration documents?

I see no mention of breaking changes between 2.2 and 3.1 doc and this is a breaking change between migrating between those versions as we have to stop using .NET Standard for a lot of our supporting library which kind of defeats one of the purposes of .NET Standard.

https://docs.microsoft.com/en-us/dotnet/core/compatibility/2.2-3.1

**Comment by Tratcher** [comment]
All of AspNetCore moved from netsteadard2.0 to netcoreapp3.x, not just this package.
https://github.com/aspnet/Announcements/issues/324
https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#migrate-libraries-via-multi-targeting

So I have to ask, how are you using just the JwtBearer package from a library? This package depends on the rest of AspNetCore.

**Comment by CreepyGnome** [comment]
That generic statement about ASP.NET Core is general, not specific to the libraries that were not multi-targeted and didn't list the libraries impacted either. As we search by the name of a class, method, package name to find what is broken in a migration doc.

We had to support .NET Standard projects there were shared across all the ASP.NET Core projects... they just where "infrastructure" stuff and since .NET Standard was supported we used it as for libraries that are not required to be executable we like to make the .NET Standard whenever possible. 

So this was just a surprise and not real reason could be found why something that was only .NET Standard went to full .NET Core instead of going multi-targeted.

It is what it is, I think the migration documents could have been better in this area by listing the ones that changed from multi-target to single target and the ones that went from single-target .NET Standard to be a single-target to .NET Core
